WalletWasabi/Wallets/Exchange/ExchangeRateProvider.cs

```diff
@@ -28,6 +29,12 @@ public async Task<ExchangeRate> GetExchangeRateAsync(string providerName, string
 		{
 			throw new NotSupportedException($"Exchange rate provider '{providerName}' is not supported.");
 		}
+
+		if(providerInfo.Name is "None" or "")
+		{
+			return new ExchangeRate("USD", -1);
+		}
```

Comment:
This tells me that the refactoring in `FeeRateProvider` was correct because we don't need this kind of hacks there.